src/App.js

```diff
@@ -1,16 +1,42 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  //  var mylist = ["Ayy", "whats", "good"]
+  //  const mapper = list.map(x : x + "\n")
   render() {
+    let mylist = ['Ayy', 'whats', 'good']
```

Comment:
`const` is preferred to `let` unless you explicitly need to reassign a variable

https://github.com/airbnb/javascript#references--prefer-const